backend/tracim_backend/templates/mail/new_upload_event_body_html.mak

```diff
@@ -0,0 +1,18 @@
+<p>${_('Hello,')}</p>
+<p>${_('{username} shared some files with you in  sharespace {workspace_name}:').format(username=uploader_username, workspace_name=workspace.label)}</p>
```

Comment:
two space between 'you in' and 'sharespace'

Reply:
I think its better to says `{username} shared some files in sharespace {workspace_name}` because all members not know who is {username}.

Reply:
Wrong name `sharespace`, its `shared space`.

Reply:
fixed.